tests/integration/sentinel/schedules/due-tasks.spec.js

```diff
@@ -89,7 +89,7 @@ const contacts = [
     reported_date: reportedDate,
   },
   {
-    _id: 'patient2',
+    _id: 'patient_2',
```

Comment:
Does adding a scenario where the `patient` doesn't have `patient_id` make sense?